test-cases/features/elasticity/scalein-scaleout-elasticity-90-percent.yaml

```diff
@@ -0,0 +1,42 @@
+test_duration: 3000
```

Comment:
50 hours?

Looks too big.
Jenkins test runs may hang, so, better to reduce it